maintenance/maintenance.html

```diff
@@ -100,37 +100,61 @@ <h2 class="h3-header">We Are Undergoing Maintenance</h1>
       </div>
     </div>
 
-    <footer class="mofo-footer">
+    <div class="donate-banner dark-theme py-5">

```

Comment:
copied markups from `footer.html`

Reply:
I assumed we wouldn't wanna load javascript on this page so I didn't include `<div class="join-us mb-5" data-layout="side-button"></div>` here as it serves as React component inject target.

Reply:
yeah, when we're down for maintenance, we don't want a component that needs to talk to the server.